tests/test_format.py

```diff
@@ -62,6 +62,7 @@
 ]
 
 PY310_CASES: List[str] = [
+    "starred_for_target",
```

Comment:
I don't think this needs to be 310-specific

Reply:
Since the PEP 617 was opt-out in 3.9, the given examples are rejected by the official AST when the user starts the python interpreter with `-Xoldparser`. It is why I've added tests to 3.10 group.